tests/algorithms/column_pooler_test.py

```diff
@@ -101,7 +101,7 @@ def testLearnSinglePattern(self):
     self.init()
 
     object = self.generateObject(1)
-    self.learn(object, numRepetitions=1, newObject=True)
+    self.learn(object, numRepetitions=2, newObject=True)
```

Comment:
Thanks - makes sense. As a rule I don't think we should override parameters unless we are specifically testing their impact.